### ISSUE TITLE
[Docs] Build out Learning Move section

### ISSUE THREAD

**Issue by clay-aptos** [aptos-labs/aptos-core#5774]
# Aptos Documentation Issue

See this issue for history:
https://github.com/aptos-labs/aptos-core/issues/4748

In short, we need to:

1. Update our existing NFT tutorial to reflect the changes made in:
https://github.com/aptos-labs/aptos-core/pull/5714
2. Convert our internal Move Workshop #1 on creating an NFT to external documentation.

**Comment by clay-aptos** [comment]
> I just want it to be clear that this work is NOT to update the current "your first NFT" tutorial but to create a new section in aptos.dev for 'Learning Move'

Please work with @movekevin to define your desires here as we already have this section of docs:
https://aptos.dev/guides/move-guides/aptos-move-guides

**Comment by clay-aptos** [comment]
Also, are you saying we don't need to update the existing Your First NFT tutorial to reflect @0xchloe 's changes? If we do, let's track that here too. If not, I will remove the item.

**Comment by clay-aptos** [comment]
Finally, @cathyaptos to expedite your request please have the team draft new workshop materials in GitHub for all to benefit rather than in internal docs that must be merged in later.

**Comment by 0xchloe** [comment]
Ah yeah, agree with cathy that we intend to create a new `Learn Move` section instead of Your First NFT. I guess using the existing `Move on Aptos` could work too? But it seems like `Move on Aptos` is more on Move concepts and less like hands-on tutorials, except the `Your First Move Module` section. @movekevin @cathyaptos wdyt? Should we put the new tutorials under `Move on Aptos` or create a new section `Learn Move`?
Also, we don't need to update the existing Your First NFT tutorial because that's targeting the SDK developers, and this tutorial is more targeting smart contracts developers.

**Comment by cathyaptos** [comment]
I think we should put it under Move on Aptos but have its new subsection like "Move Tutorials" or "Learn Move via Tutorials" or something :)

**Comment by cathyaptos** [comment]
@movekevin lmk if you agree!

**Comment by cathyaptos** [comment]
yeah. i like that as a new section for all the move tutorials we’ll plan to
make

**Comment by clay-aptos** [comment]
Please see progress in:
https://github.com/aptos-labs/aptos-core/pull/5817

And note Move on Aptos now contains only one conceptual doc: How Base Gas Works
We can certainly create a Tutorials subsection there or similar when we have many of them.

Here is a TODO from the PR: Find out why I get "No collectibles yet." Confirm the private key is emitted in the output above as it says account. If not from there, where do they derive this essential ingredient?

**Comment by cathyaptos** [comment]
Amazing! @0xchloe in the new section should we actually have the materials from workshop #2 go before the materials for workshop #1 since today's workshop was Move basics?

**Comment by cathyaptos** [comment]
To be clear :) We can publish workshop #1 first but when we publish workshop #2 let's make sure it's higher up in the order 

(aka. no change to current workflow, just commenting on the where to publish it)

**Comment by clay-aptos** [comment]
> To be clear :) We can publish workshop #1 first but when we publish workshop #2 let's make sure it's higher up in the order
> 
> (aka. no change to current workflow, just commenting on the where to publish it)

Makes sense to me. Will keep you posted.

**Comment by clay-aptos** [comment]
I am noting @perryjrandall requested a diagram during workshop #1 depicting the relationship of the various accounts (admin, source, and resource) and smart contracts.

**Comment by clay-aptos** [comment]
@movekevin, we describe admin and source accounts in the Move Workshop #1 @0xchloe presented, yet we don't describe them in:
https://aptos.dev/concepts/accounts/

Should we? We instead say:

There are two types of accounts in Aptos:

Standard account - This is a typical account corresponding to an address with a corresponding pair of public/private keys.
Resource account - An autonomous account without a corresponding private key used by developers to store resources or publish modules on chain.

**Comment by movekevin** [comment]
I don't think we need to call that out on the accounts page. That's internal to how resource accounts work and are created. We should add those details on the specific resource account page/guide.

**Comment by clay-aptos** [comment]
Noting progress on the NFT front in:
https://github.com/aptos-labs/aptos-core/pull/6266

We should follow this up with an NFT index that summarizes all docs in this section after the Mint Tool and property map docs land. Putting here so the team can help me remember.

**Comment by clay-aptos** [comment]
Hi team, I believe I have addressed all feedback in PR#5872 and then abandoned/recreated it with:
https://github.com/aptos-labs/aptos-core/pull/6296

As the former was woefully out of sync with main. Please take a quick look and approve so we don't run into more conflicts. Thanks, all.

**Comment by clay-aptos** [comment]
Hi team,

All that remains here is:
https://github.com/aptos-labs/aptos-core/pull/6202

Once that is in, I am shifting to more Move docs in:
https://github.com/aptos-labs/aptos-core/issues/6375

Please let me know how to help. Thanks!

**Comment by clay-aptos** [comment]
Hi @areshand and team,

Thanks for getting the airdrop and propertymap docs in. I am now integrating and announcing them in:
https://github.com/aptos-labs/aptos-core/pull/6579

Please see them staged at:
https://deploy-preview-6579--aptos-developer-docs.netlify.app/whats-new-in-docs/
https://deploy-preview-6579--aptos-developer-docs.netlify.app/concepts/coin-and-token/airdrop-aptos-tokens/
https://deploy-preview-6579--aptos-developer-docs.netlify.app/concepts/coin-and-token/propertymap-offchain/

And let me know if you feel they are misplaced. Thanks!
